Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs

```diff
@@ -321,19 +315,11 @@ private async Task CacheResponse(HttpResponseInfo response, string responseCache
         {
             _fileSystem.CreateDirectory(_fileSystem.GetDirectoryName(responseCachePath));
 
-            using (var responseStream = response.Content)
+            using (var fileStream = _fileSystem.GetFileStream(responseCachePath, FileOpenMode.Create, FileAccessMode.Write, FileShareMode.None, true))
```

Comment:
Well, that thing in itself was probably trying to be smart and fast for stuff gotten over network or hdd, so that if there are slow clients they won't thrash hdd head... But usually this is solved by reverse proxy in between.